src/components/HelpButton.vue

```diff
@@ -0,0 +1,155 @@
+import MenuLinks from './MenuLinks.vue';
+
+// config
+const rideToWorkByBikeConfig: ConfigGlobal = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG
+);
+setCssVar('primary', rideToWorkByBikeConfig.colorPrimary);
+
+export default defineComponent({
+  name: 'HelpButton',
+  components: {
+    MenuLinks,
+    ListFaq,
+    ContactForm,
+    DialogStates,
+  },
+  props: {
+    color: {
+      type: String as () => 'grey-10' | 'primary',
+      default: 'grey-10',
+    },
+    size: {
+      type: String,
+      default: '11px',
+    },
+  },
+  setup() {
+    const isDialogOpen = ref(false);
+
+    return {
+      isDialogOpen,
+    };
+  },
+});
+</script>
+
+<template>
+  <q-btn
+    unelevated
+    round
+    :size="size"
+    :color="color"
+    @click.prevent="isDialogOpen = true"
+    data-cy="button-help"
+  >
+    <q-icon name="question_mark" color="white" data-cy="icon-help" />
+  </q-btn>
+  <!-- Dialog -->
+  <dialog-states v-model="isDialogOpen" data-cy="dialog-help">
+    <template #title="{ state }">
+      <span v-if="state === 'default'">
+        {{ $t('index.help.titleStateDefault') }}
+      </span>
+      <span v-if="state === 'form'">
+        {{ $t('index.help.titleStateContact') }}
+      </span>
+    </template>
+    <template #content="{ state, setState, reset }">
+      <div v-if="state === 'default'">
+        <!-- FAQ for pariticipants -->
+        <list-faq
+          :title="$t('index.help.titleParticipants')"
+          variant="participant"
+        />
+        <!-- FAQ for Company coordinators -->
+        <list-faq
+          :title="$t('index.help.titleCoordinators')"
+          variant="coordinator"
+          class="q-mt-xl"
+        />
+        <!-- Section: App Guide -->
+        <div class="q-px-md q-mt-xl">
+          <h4 class="text-h5 text-weight-bold q-my-none" data-cy="title-guide">
+            {{ $t('index.help.titleGuide') }}
+          </h4>
+          <!-- Button: Replay guide -->
+          <q-btn
+            rounded
+            color="black"
+            unelevated
+            outline
+            :label="$t('index.help.buttonGuide')"
+            class="q-mt-md"
+            data-cy="button-guide"
+          />
+        </div>
+        <!-- Section: Contact us via form modal dialog -->
+        <div class="q-px-md q-mt-xl">
+          <h4
+            class="text-h5 text-weight-bold q-my-none"
+            data-cy="title-contact"
+          >
+            {{ $t('index.help.titleContact') }}
+          </h4>
+          <!-- Button: Switch to contact form -->
+          <q-btn
+            rounded
+            color="black"
+            unelevated
+            :label="$t('index.help.buttonContact')"
+            class="q-mt-md"
+            data-cy="button-contact"
+            @click.prevent="setState('form')"
+          />
+        </div>
+        <!-- Section: Useful links -->
+        <menu-links :title="$t('index.help.titleLinks')" variant="useful" />
```

Comment:
Please set `MenuLinks` component `variant` prop **default value** e.g. `default='useful'` and you don't need use prop value here `variant="useful"`.

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/601bb02996a91b9535db73380748e357fef650ca/src/components/MenuLinks.vue#L37-L40